### ISSUE TITLE
Has keras this kind of interface"flags = tf.app.flags FLAGS = flags.FLAGS flags.DEFINE_float('learning_rate', 0.01, 'Initial learning rate.')" as in tensorflow?

### ISSUE THREAD

**Issue by IvyGongoogle** [keras-team/keras#3750]
Using tensorflow to construct a model is a little more complicated, so I want to use keras to construct. But I have not found a interface as in tensorflow:
"flags = tf.app.flags
FLAGS = flags.FLAGS
flags.DEFINE_float('learning_rate', 0.01, 'Initial learning rate.')
flags.DEFINE_integer('max_steps', 2000, 'Number of steps to run trainer.')
flags.DEFINE_integer('hidden1', 128, 'Number of units in hidden layer 1.')
flags.DEFINE_integer('hidden2', 32, 'Number of units in hidden layer 2.')
flags.DEFINE_integer('batch_size', 100, 'Batch size.  '
                     'Must divide evenly into the dataset sizes.')
flags.DEFINE_string('train_dir', 'data', 'Directory to put the training data.')
flags.DEFINE_boolean('fake_data', False, 'If true, uses fake data '
                     'for unit testing.')
"
I think this kind of interface is convenient to modify some parameters.

**Comment by IvyGongoogle** [comment]
@probot-stale[bot]  thank you~~

**Comment by IvyGongoogle** [comment]
@stale. OK thanyou
